queries/raffleQueries.js

```diff
@@ -39,21 +39,27 @@ returning *`,
   return newRow;
 };
 
-const addNewParticipant = async (args, id) => {
+const addNewParticipant = async (args) => {
   const newRow = await db.oneOrNone(
-    `insert into participants(first_name, last_name, email, phone, raffle_id) values ($1, $2, $3, $4, (select id from raffle where winner_id is NULL and id = $5))
-returning *`,
+    `insert into participants(first_name, last_name, email, phone) values ($1, $2, $3, $4) on conflict do nothing returning *`,
     [
       args["first_name"].trim(),
       args["last_name"].trim(),
       args["email"],
       args?.phone || null,
-      id,
     ],
   );
   return newRow;
 };
 
+const addNewRaffleParticipant = async (id, args) => {
+  const newRow = await db.oneOrNone(
+    `insert into participants_raffle(raffle_id, participant_id) values ($1, (select id from participants where email = $2)) on conflict do nothing returning *`,
```

Comment:
would rather refactor so we don't have to look up the participant id on the fly like this